$OpenBSD: patch-remmina-plugins_vnc_libvncserver_libvncclient_rfbproto_c,v 1.1 2012/02/24 13:48:26 ajacoutot Exp $

XXX
rfbproto.c:573: error: 'rfbARD' undeclared (first use in this function)

--- remmina-plugins/vnc/libvncserver/libvncclient/rfbproto.c.orig	Fri Feb 24 12:33:42 2012
+++ remmina-plugins/vnc/libvncserver/libvncclient/rfbproto.c	Fri Feb 24 12:34:16 2012
@@ -59,6 +59,10 @@
 #include <gcrypt.h>
 #endif
 
+#ifndef rfbARD
+#define rfbARD 30
+#endif
+
 #include "minilzo.h"
 #include "tls.h"
 
